packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -9,7 +9,7 @@ describe('Slider', () => {
     Component: Slider,
     displayName: 'Slider',
     primarySlot: 'input',
-    disabledTests: ['kebab-aria-attributes'],
+    disabledTests: ['kebab-aria-attributes', 'component-has-static-classname'],
```

Comment:
this is in prep for moving over to the classnames object that sean is working on.